docs/modules/ROOT/pages/how-tos/update_maintenance.adoc

```diff
@@ -0,0 +1,23 @@
+= Maintenance and Update of an OpenShift 4 cluster
+
+. Get list of available updates:
++
+[source,console]
+----
+kubectl --as cluster-admin get clusterversion version -o json | jq '.status.availableUpdates[] | {image: .image, version: .version}'
+----
+       
+. Update the configuration hierarchy
++
+Set the following parameters to the values retrieved in the previous step:
++
+* `parameters.openshift4_version.spec.desiredUpdate.image`
+* `parameters.openshift4_version.spec.desiredUpdate.version`
+
+. Compile the cluster catalog
+
+. Enjoy the show
++
+Let the OpenShift operators do their job.
+
+See also https://docs.openshift.com/container-platform/latest/updating/updating-cluster-between-minor.html[Updating a cluster between minor versions] of the OpenShift 4 documentation.
```

Comment:
How do I enjoy the show?
The article linked does not really tell how I can follow the update. yes it mentions the machine config operator, but what do I need to look out for? How can I verify whether the upgrade worked and is finished or failed? Or are these things of the past?

Reply:
I do lack the experience to write down something that works. My intention was to improve later.

Reply:
Ok. Maybe add the line `kubectl --as cluster-admin get clusterversion version --watch` for the moment, as it displays at least whether it's progressing, faulting or done. And a nice sentence

Reply:
Done.